src/tests/http-s3.test.ts

```diff
@@ -35,4 +35,33 @@ describe("Http/s3 interface", () => {
       1235865600000,
     ]);
   });
+
+  it("allows reading from s3", async () => {
+    await connection.executeIterator(`SET s3_region='us-east-1'`);
+    await connection.executeIterator(`SET s3_access_key_id='${process.env.AWS_ACCESS_KEY_ID}'`);
+    await connection.executeIterator(`SET s3_secret_access_key='${process.env.AWS_SECRET_ACCESS_KEY}'`);
+    // NOTE: The "=" sign had to be encoded to "%3D" to make this work, 
+    //       probably something that DuckDB should be doing!
+    const result = await connection.executeIterator(
+      "SELECT * FROM parquet_scan('s3://amazon-reviews-pds/parquet/product_category%3DBooks/part-00000-495c48e6-96d6-4650-aa65-3c36a3516ddd.c000.snappy.parquet') LIMIT 1",
```

Comment:
so the uri should be url encoded?

Reply:
I created a separate DuckDB PR for fixing that on DuckDB side. Please see:
https://github.com/duckdb/duckdb/pull/2233

Reply:
cool 👍